src/sentry/http.py

```diff
@@ -165,13 +169,13 @@ def fetch_file(
 
     logger.debug("Fetching %r from the internet", url)
 
-    with SafeSession() as http_session:
-        response = None
+    with contextlib.ExitStack() as ctx:
+        http_session = ctx.enter_context(SafeSession())
 
         try:
-            try:
-                start = time.time()
-                response = http_session.get(
+            start = time.monotonic()
```

Comment:
fixed a microbug: `time.time()` means this function fails in weird ways during clock skew / leap seconds / etc.